packages/metrics/package.json

```
@@ -42,8 +13,52 @@
     "line gap",
     "leading"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/seek-oss/capsize.git",
+    "directory": "packages/metrics"
+  },
   "license": "MIT",
-  "dependencies": {},
+  "author": {
+    "name": "Michael Taranto",
+    "homepage": "https://github.com/michaeltaranto"
+  },
+  "exports": {
+    ".": {
+      "types": {
+        "import": "./dist/index.d.mts",
+        "require": "./dist/index.d.ts"
+      },
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    },
+    "./entireMetricsCollection": {
+      "types": {
+        "import": "./dist/entireMetricsCollection.d.mts",
+        "require": "./dist/entireMetricsCollection.d.ts"
+      },
+      "import": "./dist/entireMetricsCollection.mjs",
+      "require": "./dist/entireMetricsCollection.cjs"
+    },
+    "./package.json": "./package.json",
+    "./*": "./entireMetricsCollection/*"
```

```suggestion
    "./*": "./entireMetricsCollection/*.js"
```

This might need experimenting if types don't work